File: SMI-SDK/.NET SDK/Library/sdb/smi/Entry.cs
using System;
using System.Collections.Generic;
namespace pt.sapo.sdb.smi
{

    public class Entry
    {
        public String description;
        public SeverityType severity;
        public double offset;
        public double duration;
        private List<Entry> entries;
        public IDictionary<String, Object> properties;
        public int messageNumber;
        public Exception exception;
        public String linkedTraceId;

        public List<Entry> getEntries()
        {
            if (entries == null)
                entries = new List<Entry>();
            return entries;
        }
    }
}

File: SMI-SDK/Java SDK/src/pt/sapo/sdb/trace/Entry.java
package pt.sapo.sdb.trace;

import java.util.ArrayList;
import java.util.List;
import java.util.Map;

/**
 *
 * @author GIS Team
 */
public class Entry {
    public String Description;
    
    public SeverityType Severity;
    
    public double Offset;
    
    public double Duration;
    
    public List<Entry> Entries = new ArrayList<Entry>();;    
    
    public Map<String,Object> Properties;
    
    public Integer MessageNumber;   
    
    public Exception Exception;
    
    public String LinkedTraceId;    
}
